lib/services/local/trust.go

```diff
@@ -32,12 +36,14 @@ import (
 // is using local backend
 type CA struct {
 	backend.Backend
+	log *logrus.Entry
 }
 
 // NewCAService returns new instance of CAService
 func NewCAService(b backend.Backend) *CA {
 	return &CA{
 		Backend: b,
+		log:     logrus.WithFields(logrus.Fields{trace.Component: "CA"}),
```

Comment:
slog?

Reply:
I wonder if we should expose some logger in `backend.Backend` for things like this.